tools/table_validator/venv/bin/activate

```diff
@@ -0,0 +1,76 @@
+# This file must be used with "source bin/activate" *from bash*
```

Comment:
venv/* files should not be commit. Remove this file (git rm -rf venv) and add the venv files to a .gitignore

Reply:
I thought I removed that but I can remove it again